### PR TITLE
Chore/update ember

### LINKED ISSUE #29: update ember to 2.18
This could server as a trial run for updating the Hub apps.

I just did this in ember-cli-cedar, and it was delightful thanks to:

https://www.npmjs.com/package/ember-cli-update
https://github.com/ember-cli/ember-modules-codemod

We should also switch over to yarn: https://github.com/Esri/ember-cli-cedar/pull/75/commits/5f59191209890d2938bd6b8e298dff7a03d92033

I'd like to do this _after_ #27

### PR DESCRIPTION
**Pull request by tomwayson**
resolves #29 

cc @mjuniper

### CODE REVIEW

File: app/components/topic-feature/component.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-  featureService: Ember.inject.service(),
-  feature: [],
+export default Component.extend({
+  featureService: service(),
 
-  didInsertElement () {
-    this._super.apply(this, arguments);
+  init () {
+    this._super(...arguments);
+    this.set('feature', []);
```
Comment (MarkhamShofner) [inline app/components/topic-feature/component.js]: Why is `this.set('feature', []);` preferable to `feature: [],`

Comment (tomwayson) [reply app/components/topic-feature/component.js]: I had to make that change to get lint to pass, see https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/avoid-leaking-state-in-components.md

File: README.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
```
Comment (jgravois) [inline README.md]: ^^

File: README.md [hunk 3]
```diff
@@ -1,4 +1,8 @@
...
 # MyStreet
+=======
+# my-app
+>>>>>>> de5b992... message
```
Comment (jgravois) [inline README.md]: ^^

File: README.md [hunk 4]
```diff
@@ -102,3 +106,5 @@ If you have any questions, please reach out to mshofner@esri.com, or make a PR/I
 
 [](Esri Tags: ArcGIS Web Mapping Leaflet JavaScript Hub Government)
 [](Esri Language: JavaScript Ember)
+=======
+>>>>>>> de5b992... message
```
Comment (jgravois) [inline README.md]: ^^

**Comment by MarkhamShofner** [comment]
My errors have been resolved. The app is working locally. LGTM except for the little pieces John identified above.

**Comment by jgravois** [comment]
💚

**Comment by tomwayson** [comment]
@mjuniper 

Some things to watch out for:
- ember-cli-update wipes out bower.json _and_ the related .travis.yml steps, but we'll probably still need at least the former in Hub
- the new linting rules don't seem to like `this._super.apply(this, arguments);` they want     `this._super(...arguments);`
- update torii-provider-arcgis https://github.com/Esri/MyStreet/pull/36/commits/70055c3049165d7d704b439d3b7437583edc664d

**Pull request merged**